platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -161,14 +173,29 @@ BundleContext getBundleContext() {
         return warnings;
     }
 
+    private Collection<Describable> listMigratableDescribables() {
```

Comment:
Instead of building the service list by hand, we should use blueprint's `reference-list` and `reference-listener`.